CMakeLists.txt

```diff
@@ -1302,6 +1302,9 @@ check_symbol_exists(getrlimit      "${CURL_INCLUDES}" HAVE_GETRLIMIT)
 check_symbol_exists(setlocale      "${CURL_INCLUDES}" HAVE_SETLOCALE)
 check_symbol_exists(setmode        "${CURL_INCLUDES}" HAVE_SETMODE)
 check_symbol_exists(setrlimit      "${CURL_INCLUDES}" HAVE_SETRLIMIT)
+check_symbol_exists(bzero          "${CURL_INCLUDES};strings.h" HAVE_BZERO)
+check_symbol_exists(explicit_bzero "${CURL_INCLUDES};strings.h" HAVE_EXPLICIT_BZERO)
+check_symbol_exists(memset_s       "${CURL_INCLUDES};string.h" HAVE_MEMSET_S)
```

Comment:
It's safe to enclose these check inside `if(NOT WIN32)`. Saves time.

(libssh2 does this.)

Reply:
I think `${CURL_INCLUDES};` is unnecessary for these checks.

Reply:
I think so too, but we use `${CURL_INCLUDES}` on all the other checks where it's likely not needed so I went for consistency. Maybe a cleanup patch taking all checks into consideration would be in order?

Reply:
It's though because each of them needs to be carefully validated. I didn't dare touching it yet, though it's on my TODO list. For these new ones, we can be sure it's the only headers needed.

Reply:
> It's tough because each of them needs to be carefully validated. I didn't dare touching it yet, though it's on my TODO list. For these new ones, we can be sure it's the only headers needed.

This has landed since via in 8e345057761a8f796403923a96f2c8fd3edca647 #15164 .